### PR TITLE
ignore hpa warnings while deploying from samson

### PR DESCRIPTION
**Pull request by ankilosaurus**
@zendesk/compute 

Jira: https://zendesk.atlassian.net/browse/PAAS-3630

### CODE REVIEW

File: plugins/kubernetes/app/models/kubernetes/api/pod.rb [hunk 1]
```diff
@@ -3,7 +3,8 @@ module Kubernetes
   module Api
     class Pod
       INIT_CONTAINER_KEY = :'pod.beta.kubernetes.io/init-containers'
-      INGORED_AUTOSCALE_EVENT_REASONS = %w[FailedGetMetrics FailedRescale].freeze
+      INGORED_AUTOSCALE_EVENT_REASONS = %w[FailedGetMetrics FailedRescale, FailedGetResourceMetric,
+        FailedGetExternalMetric, FailedComputeMetricsReplicas].freeze
```
Comment (grosser) [inline plugins/kubernetes/app/models/kubernetes/api/pod.rb]: %w and , don't work

... use `["Foo", "Bar"]`

Comment (uthark) [reply plugins/kubernetes/app/models/kubernetes/api/pod.rb]: Also, let's split them one by line.

**Review by uthark — APPROVED** [review]

File: plugins/kubernetes/app/models/kubernetes/api/pod.rb (hunk 1 above)
Comment (aksgithub) [reply plugins/kubernetes/app/models/kubernetes/api/pod.rb]: already updated

File: plugins/kubernetes/app/models/kubernetes/api/pod.rb [hunk 2]
```diff
@@ -3,7 +3,11 @@ module Kubernetes
   module Api
     class Pod
       INIT_CONTAINER_KEY = :'pod.beta.kubernetes.io/init-containers'
-      INGORED_AUTOSCALE_EVENT_REASONS = %w[FailedGetMetrics FailedRescale].freeze
+      INGORED_AUTOSCALE_EVENT_REASONS = ["FailedGetMetrics",
```
Comment (grosser) [inline plugins/kubernetes/app/models/kubernetes/api/pod.rb]: 2-space indent plz
```suggestion
      INGORED_AUTOSCALE_EVENT_REASONS = [
        "FailedGetMetrics",
```

File: plugins/kubernetes/app/models/kubernetes/api/pod.rb [hunk 3]
```diff
@@ -3,7 +3,8 @@ module Kubernetes
   module Api
     class Pod
       INIT_CONTAINER_KEY = :'pod.beta.kubernetes.io/init-containers'
-      INGORED_AUTOSCALE_EVENT_REASONS = ["FailedGetMetrics",
+      INGORED_AUTOSCALE_EVENT_REASONS = [
+        "FailedGetMetrics",
```
Comment (uthark) [inline plugins/kubernetes/app/models/kubernetes/api/pod.rb]: Formatting?

**Review by uthark — APPROVED** [review]

**Review by grosser — APPROVED** [review]

**Pull request merged**